Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm

```diff
@@ -1213,6 +1213,63 @@ static _WKWebExtensionContextError toAPI(WebExtensionContext::Error error)
     m_backgroundWebView = nil;
 }
 
+void WebExtensionContext::loadBackgroundPageIfNecessary(CompletionHandler<void()>&& completionHandler)
+{
+    if (!extension().backgroundContentPath() || extension().backgroundContentIsPersistent()) {
```

Comment:
So this isn't called on start-up? The method name sounds a bit like something that would be called at startup as well as when needed.

Reply:
I believe it's only called when we need to perform some tasks but the background page might be asleep so we wake it up. It's currently not being used yet but it seems like something we'll need for `runtime.getBackgroundPage()` and `runtime.sendMessage()`.

Reply:
I can change it to `wakeUpBackgroundContent()` but I'm not sure if that's any better